lib/utils/moduleAlias.js

```diff
@@ -0,0 +1,38 @@
+// Require our actual `package.json` data for use by this file.
+//
+// NB: `package` is a reserved word
+const packageData = require('../../package.json')
+
+// Extracts and returns the `module-alias` configuration from our `package.json` data.
+//
+// We don't control the `_moduleAliases` variable name, we just work with it.
+// eslint-disable-next-line no-underscore-dangle
+const getModuleAliasesFromPackage = () => packageData._moduleAliases || {}
+
+// Loads aliases from our `package.json` data and returns them, transformed into ESLint's
+// configuration format: `[[alias, destination]]`.
+const getModuleAliasesForESLint = () => {
+	const moduleAliases = getModuleAliasesFromPackage()
+	return Object.keys(moduleAliases).map((stem) => [stem, moduleAliases[stem]])
+}
+
+// Loads aliases from our `package.json` data and returns them, transformed into Jest's
+// configuration format: `{ alias: destination }`.
+const getModuleAliasesForJest = () => {
```

Comment:
I just want to note that deep down I know we both really want there to be tests for these but I appreciate that WE MUST SACRIFICE THAT CONCEPT IN THE NAME OF PROGRESS.

Reply:
Yeah… I actually wasn't blocked on that by progress but by the fact that this function doesn't take input (intentionally), and instead relies on pulling live `package.json` data from the outer scope (again, by design). I… wasn't sure how to test that, and didn't want to spin wheels to figure it out (yes inthenameofprogress).

Feel free to open an issue, although, I also think we'd be made aware of any failures here quickly because linting and/or testing would break.